### DISCUSSION TITLE
SonsuzUs Github Proje | SonsuzUs Github

### DISCUSSION THREAD

**Discussion by giscus** [sonsuzus/sonsuzus.github.io#99] [Site]
# SonsuzUs Github Proje | SonsuzUs Github

SonsuzUs Bilişim Projeleri ve Programlama

https://sonsuzus.github.io/posts/anasayfa/

**Comment by sonsuzus** [comment]
Github hesabınızla yorum yapabilirsiniz tüm yazılara. Yorum sistemi entegre edilmiştir. Gönderilerde markdown kullanabilirsiniz.

**Reply by sonsuzus** [reply]
Yorum sisteminde giscus kullanılmıştır.

**Reply by sonsuzus** [reply]
Yazmak programcılığın temelidir.
